content/ansible-tower-getting-started/2-cred/_index.md

```diff
@@ -103,68 +97,75 @@ MIIEpAIBAAKCAQEA2nnL3m5sKvoSy37OZ8DQCTjTIPVmCJt/M02KgDt53+baYAFu1TIkC3Yk+HK1
 -----END RSA PRIVATE KEY-----
 ```
 
-- Copy the **complete private key** (including "BEGIN" and "END" lines) and paste it into the **SSH PRIVATE KEY** field in the web UI.
+- Copy the **complete private key** (including "BEGIN" and "END" lines) and paste it into the **SSH Private Key** field in the web UI.
 
-- Click **SAVE**
+- Click **Save**
 
-Go back to the **RESOURCES -> Credentials -> Workshop Credentials** and note
-that the SSH key is not visible.
+Your new credentials have been created, go back to the **Resources -> Credentials -> Workshop Credentials** and note that the SSH key is not visible but shown as "encrypted".
 
 You have now setup credentials for Ansible to access your managed hosts.
 
-## Run Ad Hoc Commands
+## Hold on: Ansible Execution Environments Primer!
```

Comment:
we should add some links with additional details, e.g. what the hell is a Linux Container? We assume that everybody is familiar with it.